src/screens/Connectors/ConnectorMetaData/SamsungPay/SamsungPayIntegration.res

```diff
@@ -0,0 +1,141 @@
+@react.component
+let make = (~connector, ~setShowWalletConfigurationModal, ~update, ~onCloseClickCustomFun) => {
+  open APIUtils
+  open LogicUtils
+  open SamsungPayIntegrationUtils
+  let getURL = useGetURL()
+  let fetchDetails = useGetMethod()
+  let (merchantBusinessCountry, setMerchantBusinessCountry) = React.useState(_ => [])
+  let (screenState, setScreenState) = React.useState(_ => PageLoaderWrapper.Success)
+  let formState: ReactFinalForm.formState = ReactFinalForm.useFormState(
+    ReactFinalForm.useFormSubscription(["values"])->Nullable.make,
+  )
+  let form = ReactFinalForm.useForm()
+  let {globalUIConfig: {font: {textColor}}} = React.useContext(ThemeProvider.themeContext)
+  let samsungPayFields = React.useMemo(() => {
+    try {
+      if connector->isNonEmptyString {
+        let dict =
+          Window.getConnectorConfig(connector)
+          ->getDictFromJsonObject
+          ->getDictfromDict("connector_wallets_details")
+          ->getArrayFromDict("samsung_pay", [])
+
+        dict
+      } else {
+        []
+      }
+    } catch {
+    | Exn.Error(e) => {
+        Js.log2("FAILED TO LOAD CONNECTOR CONFIG", e)
+        []
+      }
+    }
+  }, [connector])
+
+  let setSamsungFormData = () => {
+    let initalFormValue =
+      formState.values
+      ->getDictFromJsonObject
+      ->getDictfromDict("connector_wallets_details")
+      ->getDictfromDict("samsung_pay")
+      ->getDictfromDict("merchant_credentials")
+
+    form.change(
+      "connector_wallets_details.samsung_pay.merchant_credentials",
+      initalFormValue->samsungPayRequest->Identity.genericTypeToJson,
+    )
+  }
+
+  let getProcessorDetails = async () => {
+    try {
+      setScreenState(_ => Loading)
+      let paymentMethoConfigUrl = getURL(~entityName=PAYMENT_METHOD_CONFIG, ~methodType=Get)
+      let res = await fetchDetails(
+        `${paymentMethoConfigUrl}?connector=${connector}&paymentMethodType=apple_pay`,
```

Comment:
`${paymentMethoConfigUrl}?connector=${connector}&paymentMethodType=apple_pay`,

Will there be paymentMethodType = applePay?